data/hai/hai outfits.txt

```diff
@@ -359,6 +444,16 @@ outfit "Quantum Keystone"
 	"quantum keystone" 1
 	description "This chunk of rock from the mountains on the Hai homeworld is apparently nothing more than a good luck charm, but the Hai have decorated their ships with them since the height of their empire. More superstitious Hai claim that the stones somehow reduce the nausea induced by hyperspace jumps by making travel between star systems easier, and many wealthy Hai refuse to book passage on a ship without one."
 
+outfit "Value Detector"
+	category "Systems"
+	cost 260000
+	thumbnail "outfit/value detector"
+	"mass" 3
+	"outfit space" -3
+	"cargo scan power" 22
+	"cargo scan speed" 4
+	"tactical scan power" 27
```

Comment:
Is this designed to make them scan at exactly the same speed?
That would be sensible I think here.

Reply:
The power only is for the range of the scan, and you first want to know how vulnerable the ship is before learning about its cargo
You cant actually tactical scan a ship it just reveals the stats